crowbar_framework/config/application.rb

```diff
@@ -19,7 +19,7 @@
 
 module Crowbar
   class Application < Rails::Application
-    SERVER_PID = %x[ps ax | grep "rainbows master" | grep -v grep].split(" ")[0]
+    SERVER_PID = system("[ps ax | grep 'puma master' | grep -v grep].split(' ')[0]")
```

Comment:
This is used only in the support controller, we should get rid of that and replace it with some proper solution within the support controller.